src/cli/flags.go

```diff
@@ -67,6 +68,12 @@ func InitFlags() flags {
 		appName = *guess
 	}
 
+	var projectType project.ProjectType
+	tempProjectType, err := project.DetectType(".")
+	if err == nil {
```

Comment:
You could ad something like this here to improve the error message

```golang
	if err != nil {
		log.Warn(err)
	}
```

Reply:
We could do a similar thing for app-name and container-registry in a separate PR